gdp-src-build/conf/templates/raspberrypi3.local.conf

```diff
@@ -0,0 +1,43 @@
+include templates/local.inc
+
+MACHINE = "raspberrypi3"
+
+GPU_MEM = "128"
+
+CORE_IMAGE_EXTRA_INSTALL += "wayland weston"
```

Comment:
Why is this needed in the pi-specific file?  Is there any difference compared to other hardware platforms?